test/unit/spec/media/track/localtrackpublication.js

```diff
@@ -74,8 +74,8 @@ const sinon = require('sinon');
         ret = localTrackPublication.unpublish();
       });
 
-      it('should call the unpublish callback (with the LocalTrackPublication) that is passed to the constructor', () => {
-        sinon.assert.calledWith(unpublish, localTrackPublication);
+      it('should call the unpublish callback (with the LocalTrack) that is passed to the constructor', () => {
```

Comment:
Hmm, should we keep this as-is and just let the component that handles the `unpublish` command reach into the `track` property?